submissions/html/assignment1/2203031080015/Aq2.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Document</title>
+    </head>
+
+    <body>
+        <fieldset id="fieldset">
+            <legend>Product Selection Form</legend>
+            
+        <form action="#">
+            <label for="color-selection">Choose Color</label>
+            <input type="color" name="color-selection">
+            <br>
+            <br>
+            <label for="size-selection">Select size:</label>
+            <input type="range" name="size-selection" min="10" max="50" value="25" >
+            <br><br>
+            <label for="select-quantity">Select Quantity</label>
+            <input type="number" min="2" max="10" placeholder="5">
```

Comment:
Redo the PR using a fresh branch. The current version completely messes up other's file paths